src/clients/nfs-client4part.rb

```diff
@@ -62,8 +62,6 @@ def main
 
         if Builtins.contains(@w_ids, Convert.to_symbol(@widget_id))
           return ToStorage(@modify_line)
-        else
-          return nil
```

Comment:
The question is if we should explicitly return `nil` in this case. That `nil` default might be confusing for someone...

What about:
```
Builtins.contains(@w_ids, Convert.to_symbol(@widget_id)) ? ToStorage(@modify_line) : nil
```
or if the line is too long:
```
is_included = Builtins.contains(@w_ids, Convert.to_symbol(@widget_id))
is_included ? ToStorage(@modify_line) : nil
```